src/workflows-app/components/SubmissionDetailsInputsTable.ts

```diff
@@ -96,6 +96,10 @@ const SubmissionDetailsInputsTable = ({ configuredInputDefinition }: SubmissionD
                         .source as RecordLookupInputSource;
                       return h(TextCell, [source.record_attribute]);
                     }
+                    if (inputTableData[rowIndex].source.type === 'literal') {
```

Comment:
I am not sure what is the state of tests for this component, but it would be good to add some tests for this.